### PR TITLE
added api for update records

### PR DESCRIPTION
**Pull request by palortoff**
Hi,

added support for update records as update is deprecated.

Thanks!

### CODE REVIEW

**Comment by edwellbrook** [comment]
@palortoff Hey! Thanks for taking the time to make these changes. I have a few questions/requests:

- what is the purpose of the `.gitignore` changes? I'm not familiar with the additions
- could you please go through and update the single quotes for strings to double quotes (just so it matches the rest of the codebase)

File: index.js [hunk 1]
```diff
@@ -251,6 +251,24 @@ class Client {
         }, callback);
     }
 
+   /**
+     * All updates within the given interval
+     *
+     * http://www.thetvdb.com/wiki/index.php?title=API:Update_Records
+     *
+     * @param {string} interval (day|week|month|all)
```
Comment (edwellbrook) [inline index.js]: could this be changed to `@param {String} interval - day|week|month|all`

File: index.js [hunk 2]
```diff
@@ -251,6 +251,24 @@ class Client {
         }, callback);
     }
 
+   /**
```
Comment (edwellbrook) [inline index.js]: this should be indented in by 1 extra space

File: index.js [hunk 3]
```diff
@@ -235,7 +235,7 @@ class Client {
     /**
      * Get series and episode updates since a given unix timestamp
      *
-     * http://www.thetvdb.com/wiki/index.php?title=API:Update_Records
+     * http://thetvdb.com/wiki/index.php?title=API:Updates
```
Comment (edwellbrook) [inline index.js]: please can the url be `http://www.thetvdb.com/wiki/index.php?title=API:Updates` (added www.) to match other urls

Comment (edwellbrook) [inline index.js]: please can the url be `http://www.thetvdb.com/wiki/index.php?title=API:Updates` (added www.) to match other urls

File: .gitignore [hunk 4]
```diff
@@ -8,3 +8,4 @@ pids
 lib-cov
 coverage
 .grunt
+.idea
```
Comment (edwellbrook) [inline .gitignore]: still not sure what `.idea` is for

**Comment by palortoff** [comment]
Hi,

thanks for the quick response!
Made the changes you asked for.

the .gitignore changes (.idea) are for project files made by my IDE (Webstorm from Jetbrains) which I did not want to commit or omit manually. It's a fairly common entry in .gitignore files.
The playground was just some file i fiddled around with. this should not be in the .gitignore. removed.

If you have any more remarks, let me know. I'll work them in tomorrow.

File: index.js [hunk 5]
```diff
@@ -251,6 +251,24 @@ class Client {
         }, callback);
     }
 
+    /**
+      * All updates within the given interval
```
Comment (edwellbrook) [inline index.js]: really sorry, but only the first line of the comment block was supposed to be indented by 1 space. (so the first stars line up) eg:

```
/**
 * blah
 * blah
 */

**Comment by edwellbrook** [comment]
following on from your comment just now — happy for you to keep `.idea` in gitingore. just fix the comment block and i'll merge, and update :)

**Comment by palortoff** [comment]
done.
Thanks!

**Comment by palortoff** [comment]
forgot to update the readme...

**Comment by edwellbrook** [comment]
was just about to update that myself. saved me a little time :p thanks

**Comment by edwellbrook** [comment]
@palortoff Thanks again (so much) for the pull request and your additions. I've just updated `node-tvdb` to version 1.2.0 on npm :smile:

**Comment by palortoff** [comment]
Awesome!
Thanks for a great package and your fast feedback!

**Pull request merged**